allUpperCase.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/bin/bash
```
This is not the shebang we agreed to use. For consistency I recommend to change it to this one.